index.js

```diff
@@ -56,9 +59,15 @@ function fastifyInjector (injectorOpts = {}, fastify = require('fastify')()) {
         // Provide access to the original plugin.
         injectPlugin[CALL_ORIGINAL] = originalPlugin
         delete injectors.plugins[pluginName]
-        return register(wrapPlugin(copyPluginEncapsulation(injectPlugin)), opts)
+        register(wrapPlugin(copyPluginEncapsulation(injectPlugin)), opts)
+      }
+
+      if (!injectPlugin) {
+        register(wrapPlugin(originalPlugin), opts)
       }
```

Comment:
[nit]: an `else` block would work here
```suggestion
      } else {
        register(wrapPlugin(originalPlugin), opts)
      }
```